sessionctx/variable/session.go

```diff
@@ -149,6 +149,9 @@ type SessionVars struct {
 	// GlobalAccessor is used to set and get global variables.
 	GlobalVarsAccessor GlobalVarAccessor
 
+	// LastFoundRows is the last found rows of previous query statement
```

Comment:
`FoundRow` should return a row instead of an integer.
`FoundRows` can be interpreted as `found number of rows`.